### PR TITLE
Enhancements with constants in forms and if-then-else

### PR DESCRIPTION
**Pull request by brunomichetti**
- Add constants names to forms
- Fix error in loginpage with login error and forgot password
- Change convention on if-then-else

### CODE REVIEW

File: src/components/targets/CreateTargetForm.js [hunk 1]
```diff
@@ -75,13 +75,13 @@ const CreateTargetForm = ({
 
   const handleChangeTopic = (select_topic) => {
     setInputs((inputs) => ({ ...inputs, topic: select_topic['value'] }));
-    setErrors(omit(errors, 'topic'));
+    setErrors(omit(errors, targetFormNames.topic));
```
Comment (manema) [inline src/components/targets/CreateTargetForm.js]: you could destructure the object at the beginning of the component, so you avoid _targetFormNames_ in all the sentences.
```suggestion
    const { topic, radius, title } = targetFormNames;
    ...
    setErrors(omit(errors, topic));
```

**Review by manema — APPROVED** [review]

File: src/components/users/EditProfileForm.js [hunk 2]
```diff
@@ -98,10 +98,10 @@ const EditProfileForm = ({ user, setEditProfile, newImg }) => {
           id: 'editprofile.update.name.text',
         })}
         inputType="text"
-        inputName="name"
+        inputName={userFormNames.name}
```
Comment (tamarasuarezrod) [inline src/components/users/EditProfileForm.js]: destructure userFormNames too

**Review by tamarasuarezrod — APPROVED** [review]

File: src/components/targets/CreateTargetForm.js (hunk 1 above)
Comment (brunomichetti) [reply src/components/targets/CreateTargetForm.js]: Great idea

**Review by manema — APPROVED** [review]

**Review by tamarasuarezrod — APPROVED** [review]
👏

**Pull request merged**